src/styles/Home.css

```diff
@@ -47,3 +47,60 @@
   width: 4vw;
   margin: 0px 1.5vw 0px 1.5vw;
 }
+
+@media (max-width: 380px){
+    .LearnMoreButton img{
+        width: 30vw;
+    }
+    .SocialMediaIcons img {
+        width: 30px;
+        margin: 0px 1.5vw 0px 1.5vw;
+    }
+}
+
+@media (max-width: 430px) and (min-width: 381px){
+    .LearnMoreButton img{
+        width: 30vw;
+    }
+    .SocialMediaIcons img {
+        width: 35px;
+        margin: 0px 1.5vw 0px 1.5vw;
+    }
+}
+
+@media (max-width: 480px) and (min-width: 431px){
+    .LearnMoreButton img{
+        width: 30vw;
+    }
+    .SocialMediaIcons img {
+        width: 35px;
+        margin: 0px 1.5vw 0px 1.5vw;
+    }
+}
+@media (max-width: 580px) and (min-width: 481px){
+    .LearnMoreButton img{
+        width: 30vw;
+    }
+    .SocialMediaIcons img {
+        width: 35px;
+        margin: 0px 1.5vw 0px 1.5vw;
+    }
+}
+
+@media (max-width: 750px) and (min-width: 581px){
+    .LearnMoreButton img{
+        width: 30vw;
+    }
+    .SocialMediaIcons img {
+        width: 35px;
+        margin: 0px 1.5vw 0px 1.5vw;
+    }
+}
+
+@media (max-width: 1800px) and (min-width: 751px){
+
+}
+
+@media (min-width: 1801px){
+
```

Comment:
Home.css: Code is clean and good use of @media queries to cater to different devices with different resolutions.